.github/workflows/auto-update.yml

```
@@ -0,0 +1,150 @@
+name: Auto Update SDK
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)